config/routes.config.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+
+const controller = require('../controllers/spotify.controllers')
```

Comment:
```suggestion
const spotifyController = require('../controllers/spotify.controllers')
```

Piensa que esto es escalable! es decir ahora solo tienes un controlador pero tendrás mas en el futuro de una aplicacion, asi que si nos acostumbramos desde ya a ser mas descriptivos con las variables, teneis mas de la mitad del curro ya hecho! y no pasa nada porque una variable sea larga no todas las variables tienen que serlo jeje